src/diffusers/models/modeling_utils.py

```diff
@@ -156,6 +157,24 @@ class ModelMixin(torch.nn.Module):
     def __init__(self):
         super().__init__()
 
+    def __getattr__(self, name):
```

Comment:
@pcuenca We need an addtitional `__getattr__` here because PyTorch's `nn.Module` also overwrites `__getattr__` so we need to catch this here first. Note here we can call `return super().__getattr__(name)` instead of `return getattr(self, name)`